i-am-terrible.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+// I'm writing this directly in the github gui, so it should be full of syntax errors, in addtion to these terrible ones I'm making on purpose      
+
+function IAMAFUNCTION(int $one) {
+$x=array('one'=>0,'one'=>0, 'one'=>0, 'one'=>0, 'one'=>0, 'one'=>0, 'one'=>0, 'one'=>0 );
+
+return $x=1;
+}
```

Comment:
__[code_sniffer]__
* Expected 1 blank line at end of file; 2 found

<!-- Comment by [SideCI](https://sideci.com) -->